src/store/slices/stakingHub/actionsAsync.ts

```diff
@@ -270,6 +270,8 @@ export const createAsyncReducer = (builder: ActionReducerMapBuilder<typeof initi
         state.safeInfo.data.registeredNodesInNetworkRegistryParsed = tmp;
         state.onboarding.nodeAddress = tmp[tmp.length - 1];
       }
+      state.safeInfo.data.allowance = action.payload.allowance;
+      state.safeInfo.data.balance = action.payload.balance;
```

Comment:
is that needed?
you already have `state.safeInfo.data = action.payload;` over it